### DISCUSSION TITLE
Is it possible to use self hosted llm model?

### DISCUSSION THREAD

**Discussion by marcin-github** [Drakkar-Software/OctoBot#2810] [General]
Hi,
I see that Octobot supports AI. Is it possible to connect to own llm model hosted on ollama? Does it make sense to use local llm instead gpt?

**Comment by Herklos** [comment]
Hi,

Yes, here is a guide https://www.octobot.cloud/en/guides/octobot-interfaces/chatgpt#custom-llm-base-url-for-prediction.

**Reply by XenomorphHun** [reply]
this guide not help to connect a locally installed LLM
we really need a complete guide because its very complex and octobot always refuse connection to any locally installed LLM what is not GPT based.

**Comment by marcin-github** [comment]
My question comes from reading this guide and the fact that I was unable to get the connection to my own llm model to work. I defined:
```
    "services": {
        "GPT": {
            "llm-custom-base-url": "http://192.168.y.x:11434/api/generate",
            "openai-secret-key": "Empty"
        },
[snip]
```

but there is no traffic to llm . I expected to see, at least, Octo verify connection to llm. But I see nothing on tcpdump. So I assumed that only Openai is supported. 
Could you give me any hint how force octobot to connect to llm to check if it works together?
